src/modules/peek/Peek.FilePreviewer/FilePreview.xaml

```diff
@@ -102,6 +102,16 @@
             LoadingState="{x:Bind UnsupportedFilePreviewer.State, Mode=OneWay}"
             Source="{x:Bind UnsupportedFilePreviewer.Preview, Mode=OneWay}"
             Visibility="{x:Bind IsUnsupportedPreviewVisible(UnsupportedFilePreviewer, Previewer.State), Mode=OneWay}" />
+
+        <TextBlock
+            x:Uid="NoMoreFiles"
+            x:Name="NoMoreFiles"
+            HorizontalAlignment="Center"
+            VerticalAlignment="Center"
+            Foreground="{StaticResource TextFillColorSecondaryBrush}"
+            Visibility="Collapsed"
```

Comment:
Binding missing?

Reply:
I removed the Text binding from the original commit because the text property is now being loaded directly from the Resources via the Uid. There is an entry in Resources for `NoMoreFiles.Text`.

Reply:
I mean the visibility property binding

Reply:
Ah, I understand. The visibility of the TextBlock is controlled directly via the `OnItemPropertyChanged` handler in `FilePreview.xaml.cs`, line 164:

```            NoMoreFiles.Visibility = NumberOfFiles == 0 ? Visibility.Visible : Visibility.Collapsed;```